Editor/AddressableImporter.cs

```diff
@@ -91,21 +91,24 @@ static AddressableAssetGroup CreateAssetGroup<SchemaType>(AddressableAssetSettin
         var guid = AssetDatabase.AssetPathToGUID(assetPath);
         var entry = settings.CreateOrMoveEntry(guid, group);
 
-        // Apply address replacement if address is empty or path.
-        if (string.IsNullOrEmpty(entry.address) ||
-            entry.address.StartsWith("Assets/") ||
-            rule.simplified ||
-            !string.IsNullOrWhiteSpace(rule.addressReplacement))
+        if (entry != null)
```

Comment:
So entry can be null? (returns by`settings.CreateOrMoveEntry(guid, group);`)

Reply:
any stanage file such as .DS_Store will cause entry be null